contracts/FlowIDTableStaking.cdc

```diff
@@ -127,21 +127,21 @@ access(all) contract FlowIDTableStaking {
 
         /// The total tokens that only this node currently has staked, not including delegators
         /// This value must always be above the minimum requirement to stay staked or accept delegators
-        access(all) var tokensStaked: @FlowToken.Vault
+        access(Identity) var tokensStaked: @FlowToken.Vault
```

Comment:
Sorry, I'm not seeing where this Identity entitlement is defined. What is the purpose of it?

Reply:
It's built in. See https://developers.flow.com/next/cadence/language/access-control#the-identity-entitlement-mapping

Reply:
This is a built-in entitlement. `Identity` entitlement mapping basically says to map any entitlement of the parent as-is to the child/field. More details here: https://github.com/onflow/flips/blob/120b8aa473e5040db2b99034d2118fb78d053cdf/cadence/20230623-entitlement-improvements.md#syntax-for-identity-mapping

We might want to revisit to see if this is the behavior we want, or whether we need to restrict this more. For the moment I used `Identity` here.